packages/openapi-v2/src/keys.ts

```diff
@@ -0,0 +1,16 @@
+// Copyright IBM Corp. 2018. All Rights Reserved.
+// Node module: @loopback/openapi-v2
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+// Note for review purpose, will be deleted when land the PR:
+// We can have a discussion for each key's name and its value:
+// I will add my proposal in a git comment under this line
+
+export namespace ControllerKeys {
+  export const METHODS_KEY = 'controller:methods';
+  export const METHODS_WITH_PARAMETERS_KEY = 'controller:methods:parameters';
+  export const PARAMETERS_KEY = 'controller:parameters';
+  export const CLASS_KEY = 'controller:class';
+  export const CONTROLLER_SPEC_KEY = 'controller:controller-spec';
+}
```

Comment:
A controller can have metadata for different api styles. It's probably better to use `openapi-v2` as the namespace. For example:
- `openapi-v2:methods`
- `openapi-v2:parameters`

Please note the controller is the target for such metadata, there is no need to add `controller` to the keys. 

Later on, we can have more keys, such as:
- `openapi-v3:methods`
- `openapi-v3:parameters`
- `grpc:methods`
- `grpc:parameters`

Reply:
Yeah a repeating of `controller` is too silly and could result in conflicts 🙊 I once named them as `swagger:*` while finally changed all to `controller:*` when thinking of we may add decorators of other classes like `Response`.
I am changing them back, the metadata target is already the best describe for its controller nature.